### ISSUE TITLE
We don't provide a dist bundle anymore

### ISSUE THREAD

**Issue by davidknezic** [axa-ch/style-guide#684]
Just recently, Pascal (Knecht) and I tried to include the Web Style Guide into his JSF project. There he needed the raw `.css`, `.js` and `icons.svg` file. But we don't provide a dist bundle anymore with all those files in it.

So we had to download the files from `https://design.axa.ch/dist/{all.min.js,all.min.css,icons.svg}`, which isn't really user friendly.

**Comment by AndyOGo** [comment]
I thought it should be in `dist/bundles/*`?
At least on my local machine I get this folder after running `npm run build`.

Our `.npmignore` file does not block them...
````sh
dist/docs/
docs/
lib/
tasks/
tmp/
.travis.yml
coffeelint.json
Dockerfile
````

**Comment by AndyOGo** [comment]
@davidknezic 
I quickly checked our axa.ch `node_modules` folder and it is there:
<img width="444" alt="screen shot 2016-11-14 at 12 17 36" src="https://cloud.githubusercontent.com/assets/914443/20262879/7b0f9ca0-aa64-11e6-9d65-6bc26221f941.png">

**Comment by AndyOGo** [comment]
@davidknezic 
Maybe we change that location to where those files ended up previously?

**Comment by davidknezic** [comment]
Yes, they are in the npm package, but we don't provide a way to download the assets for people not using npm.

**Comment by davidknezic** [comment]
We used to attach a dist archive, like in this release:
https://github.com/axa-ch/style-guide/releases/tag/v0.9.0

**Comment by AndyOGo** [comment]
@davidknezic 
Oh, I didn't know that.

**Comment by AndyOGo** [comment]
I searched the [v0.9.0](https://github.com/axa-ch/style-guide/tree/v0.9.0) of this repo and found this old gulp task https://github.com/axa-ch/style-guide/blob/v0.9.0/tasks/release-dist.coffee
